androidshared/src/main/java/org/odk/collect/androidshared/ui/multiclicksafe/MultiClickSafeMaterialButton.kt

```diff
@@ -3,23 +3,32 @@ package org.odk.collect.androidshared.ui.multiclicksafe
 import android.content.Context
 import android.util.AttributeSet
 import com.google.android.material.button.MaterialButton
+import org.odk.collect.androidshared.R
 import org.odk.collect.androidshared.ui.multiclicksafe.MultiClickGuard.allowClick
 
-open class MultiClickSafeMaterialButton : MaterialButton {
-    constructor(context: Context) : super(context)
+open class MultiClickSafeMaterialButton @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : MaterialButton(context, attrs, defStyleAttr) {
+    private val screenName: String
 
-    constructor(context: Context, attrs: AttributeSet?) : super(
-        context,
-        attrs
-    )
-
-    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
-        context,
-        attrs,
-        defStyleAttr
-    )
+    init {
+        context.theme.obtainStyledAttributes(
```

Comment:
Could we add a helper to `ContextUtils` for grabbing theme string theme attributes like this?

Reply:
Actually, kotlin offers an extension function called `withStyledAttributes` so we can use it https://github.com/getodk/collect/pull/6098/commits/ffa8c7990cbcc19879622d950c659937931726e4